README.md

```diff
@@ -6,16 +6,17 @@ Wraps the inline content with `ins` and `del` tags.
 
 Installation
 ------------
-
-    pip install git+git://github.com/aleray/mdx_del_ins.git
-
+    
+    copy mdx_del_ins_markdown_3_0_1.py as a module
```

Comment:
This seems like a regression in usability to me. Dependency management with pip is a highly desirable feature.